Fix errors like:
/usr/bin/glib-mkenums: gstrtspdefs.h:234: Failed to parse `  /*< protected >*/ '

--- a/gst-libs/gst/rtsp/gstrtspdefs.h
+++ b/gst-libs/gst/rtsp/gstrtspdefs.h
@@ -231,7 +231,7 @@
  * Enumeration of rtsp header fields.
  */
 typedef enum {
-  /*< protected >*/
+  /* < protected > */
   GST_RTSP_HDR_INVALID,
 
   /*
@@ -345,7 +345,7 @@
  * Enumeration of rtsp status codes.
  */
 typedef enum {
-  /*< protected >*/
+  /* < protected > */
   GST_RTSP_STS_INVALID                              = 0,
   GST_RTSP_STS_CONTINUE                             = 100,
   GST_RTSP_STS_OK                                   = 200,
--- a/gst-libs/gst/audio/multichannel.h
+++ b/gst-libs/gst/audio/multichannel.h
@@ -77,7 +77,7 @@
    * are defined or all positions are undefined, but can't mix'n'match  */
   GST_AUDIO_CHANNEL_POSITION_NONE,
 
-  /*< private >*/
+  /* < private > */
   /* don't use - counter */
   GST_AUDIO_CHANNEL_POSITION_NUM
 } GstAudioChannelPosition;
